### ISSUE TITLE
Add support for Visual Studio Code

### ISSUE THREAD

**Issue by fatcerberus** [spheredev/neosphere#149]
[Visual Studio Code](https://code.visualstudio.com/) is a general-purpose and extensible IDE with excellent JavaScript and TypeScript support out of the box.  This theoretically makes it a good fit for the Sphere v2 toolchain, particularly once full TypeScript integration lands in miniSphere 5.0.

A VS Code plugin will need to integrate with both miniSphere and Cell at the least, and it would nice if SSJ integration were present too so that Sphere developers could debug without leaving the IDE.

**Comment by fatcerberus** [comment]
Placed on back burner because the hypothetical plugin would still only allow code to be edited; editing other assets (maps, e.g.) will necessarily require a separate tool such as Tiled, making VS Code support something nice-to-have but not really a necessity.
